Jellyfin.ApiClient/ApiWebSocket.cs

```diff
@@ -575,15 +505,12 @@ private void OnGeneralCommand(string json)
                 }
                 if (args.KnownCommandType.Value == GeneralCommandType.DisplayMessage)
                 {
-                    string header;
-                    string text;
-                    string timeoutMs;
 
-                    args.Command.Arguments.TryGetValue("Header", out header);
-                    args.Command.Arguments.TryGetValue("Text", out text);
-                    args.Command.Arguments.TryGetValue("TimeoutMs", out timeoutMs);
+                    args.Command.Arguments.TryGetValue("Header", out string header);
+                    args.Command.Arguments.TryGetValue("Text", out string text);
+                    args.Command.Arguments.TryGetValue("TimeoutMs", out string timeoutMs);
 
-                    long? timeoutVal = string.IsNullOrEmpty(timeoutMs) ? (long?)null : long.Parse(timeoutMs, CultureInfo.InvariantCulture);
```

Comment:
Why remove the `CultureInfo.InvariantCulture`?

Reply:
This was an oops, fixed in 
93bfb86